tests/test_edgeql_explain.py

```diff
@@ -54,8 +54,12 @@ def assert_plan(self, data, shape, message=None):
             data, shape, fail=self.fail, message=message)
 
     async def explain(self, query, *, execute=True, con=None):
+        con = (con or self.con)
+        # Disable sequential scan so that we hit the index even on small
+        # datasets.
+        await con.query_single('select _set_seqscan("off")')
         no_ex = '(execute := False) ' if not execute else ''
-        return json.loads(await (con or self.con).query_single(
+        return json.loads(await con.query_single(
```

Comment:
Should we make it configurable whether we turn of seqscan? Maybe some of tests could leave it on which would allow us to reduce test churn. Maybe not really, though.

Reply:
For analyze tests leaving it on doesn't really help us, I think. If in the future we run into analyze tests that specifically need seqscan on then we can make it configurable.

Reply:
To be clear regarding the test churn, part of this is also reducing the number of tests objects created dramatically. So the results of the original tests would likely not align even if you turned the seqscan on for them.